docker/env/hydra.sh

```diff
@@ -67,8 +67,22 @@ for gid in $(id -G); do
     group_args+=(--group-add "$gid")
 done
 
+if [[ $BUILD_USER_EMAIL == *@scylladb.com ]]; then
```

Comment:
We need to have this logic stored in one place. Please run the the `get_username` from `sdcm.utils.common`.
It should be  something like this:
```bash
SCT_RUN_BY_USER=$(python3 -c "from sdcm.utils.common import get_username;print(get_username())")
```

Reply:
I don't  think its possible, since import would fail outside of the docker. you won't have any of the packages needed installed

maybe we should add hydra command to get the username

but again the reason for thus was for cleanup to kill it, we aren't doing that by user.

maybe we are better of with just the job path, and we'll  be able to track from there if needed ?

Reply:
let's put in separate module

Reply:
moved the code to a separate module which doesn't depend on any 3rd-party module.